include/openspace/rendering/renderable.h

```diff
@@ -97,7 +98,9 @@ class Renderable : public properties::PropertyOwner {
 
     RenderBin renderBin() const;
     void setRenderBin(RenderBin bin);
-    bool matchesRenderBinMask(int binMask);
+    bool matchesRenderBinMask(int binMask) const noexcept;
+
+    bool matchesSecondaryRenderBin(int binMask) const noexcept;
```

Comment:
Just curious about what the noexcept does?

Reply:
It's a specifier that essentially tells the compiler that this function won't throw any exceptions, allowing it to do further optimizations :)

Reply:
Both these checks have to be done 1-2 times every frame, for each renderable. So any way it can be made faster, the better!

Or at least that's my thought on why you suggested adding them @alexanderbock ?

Reply:
Yea, exactly.  If we start using it a bit more often, we might save a few CPU cycles here and there.  And good to start wherever we can.

For more information: https://eel.is/c++draft/except.spec#1
:P